homeassistant/components/systemmonitor/coordinator.py

```diff
@@ -38,16 +38,24 @@ class SensorData:
     def as_dict(self) -> dict[str, Any]:
         """Return as dict."""
         return {
-            "disk_usage": {k: str(v) for k, v in self.disk_usage.items()},
+            "disk_usage": {k: str(v) for k, v in self.disk_usage.items()}
+            if self.disk_usage
+            else None,
```

Comment:
We want to avoid multi-line ternaries in our codebase.

Please adjust this PR accordingly.

../Frenck